stories/Popconfirm.stories.ts

```diff
@@ -1,13 +1,12 @@
 import { CommonModule } from '@angular/common';
 import { Meta, Story } from '@storybook/angular/types-6-0';
+import { AlertModule } from 'projects/ion/src/lib/alert/alert.module';
```

Comment:
```suggestion
import { AlertModule } from '../projects/ion/src/lib/alert/alert.module';
```
This will fix the build problem on pipe.